packages/ui-toggle-details/src/ToggleGroup/index.tsx

```diff
@@ -92,23 +91,19 @@ class ToggleGroup extends Component<ToggleGroupProps> {
   }
 
   focus() {
-    // @ts-expect-error ts-migrate(2531) FIXME: Object is possibly 'null'.
-    this._button.focus()
+    this._button && (this._button as HTMLElement).focus()
```

Comment:
You can do the same typing (HTMLElement) for `this._button` here as in ToggleDetails